lib/elixir/pages/library-guidelines.md

```diff
@@ -84,7 +84,7 @@ The usage of `:ok`/`:error` tuples is about the domain that the function works o
 
 Elixir programs should prefer to validate data as close to the end user as possible, so the errors are easy to locate and fix. This practice also saves you from writing defensive code in the internals of the library.
 
-For example, imagine you have an API that receives a filename as a binary. At some point you will want to write to this file. You could have a function like this:
```

Comment:
I'm not sure adding this one change is correct.

Reply:
It's not 🙃 

https://www.pristineword.com/comma-at-some-point/